autoload/tagbar/prototypes/basetag.vim

```diff
@@ -3,6 +3,9 @@ let s:visibility_symbols = {
     \ 'protected' : '#',
     \ 'private'   : '-'
 \ }
```

Comment:
Can this assignment of `s:vilibility_symbols` be dropped entirely? It seems to be a duplicate with the same data being set in the default options array and then parsed into `s:` variables per-option. Whichever place works, I'd rather see the data in one place or the other, not both.